web/themes/custom/bglobal/source/_patterns/01-molecules/hero/hero.twig

```diff
@@ -0,0 +1,22 @@
+<section class="hero__container">
+  <div class="hero__image__container">
+    <img class="hero__image" src="{{ hero.img.src }}" alt="{{ hero.img.alt }}">
+  </div>
+  <div class="hero__text__container container">
+    <h1 class="hero__title h2">
```

Comment:
`h1` tag should not have `h2` as class here. We should probably use `h2` tag, since the page name will be `h1`.